dotcom-rendering/src/web/components/Doughnut.test.tsx

```diff
@@ -12,19 +12,23 @@ const mockSections = [
 		label: 'Dat2',
 		color: '#ce070c ',
 	},
-];
+] as const;
 
 describe('Doughnut', () => {
 	it('should display the given labels', () => {
-		const { getByText } = render(<Doughnut sections={mockSections} />);
+		const { getByText } = render(<Doughnut sections={[...mockSections]} />);
```

Comment:
Ee can't assign a readonly object to the mutable `SectionType`. There may be a cleaner way to do this?

Reply:
I have opted for destructuring `mockSections` into its consitutuents, as we’re using these in the tests explicitely.